demos/config/jenkins.yml

```diff
@@ -32,6 +32,10 @@ unclassified:
     - elastic:
         kibanaBaseUrl: "http://localhost:5601"
         name: "Elastic Observability"
+        elasticLogsBackend:
```

Comment:
https://github.com/jenkinsci/opentelemetry-plugin/pull/405 was merged with those changes , let me see what's the reason is not anymore there

Reply:
https://github.com/jenkinsci/opentelemetry-plugin/blob/c8ebe060f31319effcc260b135c6f4674f975433/demos/config/jenkins.yml#L36-L49

Reply:
I have an outdated branch